### ISSUE TITLE
Sails js Views are not loading.. 

### ISSUE THREAD

**Issue by anil-codewave** [balderdashy/sails#3344]
Hi there,

I am working on sails.js v0.11 with mongodDB. I am using simple ejs forms as my views. All my views are working fine since yesterday. When i opened in the morning the views are not loading even the sail.js homepage. All my controller actions, routes  and models are working fine, even i am getting correct Json response for all requests. The only things are not working is views.  This is the third time i am facing this problem and i am really sick of creating new project and moving all my code to newly created project. I don't know, why sails.js views are suddenly stops working after few days. please can anyone help me with this problem. 

Thanks in advance.. 

Regards
Anil

**Comment by Wizzercn** [comment]
Maybe the controller has no return res event:
return res.view('',{});
or no res.end();

All in all, the code is not easy to find a certain details of the code error

**Comment by anil-codewave** [comment]
Thanks wizzercn for your suggestion . 

I checked all my code by moving all my code to new project, there it is working fine.

i didn't change anything in my code which was there in my old project, after some days, the views won't work and this is the problem i am facing..

i really don't know, what's wrong with my sails project. 

please help me with this.

thanks in advance.

**Comment by jdcauley** [comment]
@anil-codewave are you still having issues?

Could you provide a repo and environmental details so that we might duplicated the issue?

**Comment by sailsbot** [comment]
Thanks for posting, @anil-codewave.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
